docs/ref-fleet-yaml.md

```diff
@@ -76,16 +63,7 @@ helm:
   repo: https://charts.rancher.io
 
   # The version of the chart or semver constraint of the chart to find. If a
-  # constraint is specified it is evaluated each time git changes.
-  #
-  # The version also determines which chart to download from OCI registries.
-  # Note: OCI registries don't support the '+' character, which is supported by
-  # semver.  When pushing a helm chart with a tag containing the '+' character
-  # helm automatically replaces '+' to '_' before uploading it.
-  #
-  # You should use the version with the '+' in this file, as the '_' character
-  # is not supported by semver and Fleet also replaces '+' to '_' when accessing
-  # the OCI registry.
```

Comment:
Nit: Add a comment like the one in the paragraph about the chart (see whatever-section below for more details), since some documentation has been moved, so new readers know there is more information.

Reply:
Good point, thanks! Added.
I've also taken the opportunity to backport this to v0.12, v0.11 and v0.10.